gitops/pac/.tekton

```diff
@@ -0,0 +1 @@
+../sre/template/.tekton
```

Comment:
The link should be in the other direction. Keep the code where it is, next to the other components it interacts with and references it from the template/example directory.